sickbeard/name_parser/parser.py

```diff
@@ -139,7 +139,7 @@ def _parse_string(self, name):  # pylint: disable=too-many-locals, too-many-bran
                 result = ParseResult(name)
                 result.which_regex = [cur_regex_name]
                 result.score = 0 - cur_regex_num
-                result.proper_tags = [x.raw for x in guess.get('other') if x._value=='Proper']
+                result.proper_tags = [x.raw for x in guess.get('other', '') if x._value=='Proper']
```

Comment:
result.proper_tags = [x.raw for x in guess.get('other') if x._value=='Proper']
 TypeError: 'NoneType' object is not iterable